src/LockManager.php

```diff
@@ -209,6 +221,10 @@ public function unlock(Lock $lock)
 			$this->debug('> sent unlock request to '. $this->servers[$i][0] .':'. $this->servers[$i][1]);
 		}
 		
+		if (count($this->sockets) < $this->quorum) {
```

Comment:
unlock() does writes.  This handles the scenario of the lock servers dying between the lock() and unlock().